src/main/java/com/depromeet/domain/follow/application/FollowService.java

```diff
@@ -224,6 +224,25 @@ public FollowListResponse findFollowList(Long targetId) {
                 targetMember.getProfile().getNickname(), followingList, followerList);
     }
 
+    public FollowerDeletedResponse deleteFollower(Long targetId) {
+        final Member currentMember = memberUtil.getCurrentMember();
+        final Member targetMember = getTargetMember(targetId);
+
+        MemberRelation memberRelation =
+                memberRelationRepository
+                        .findBySourceIdAndTargetId(targetMember.getId(), currentMember.getId())
+                        .orElseThrow(() -> new CustomException(ErrorCode.FOLLOW_NOT_EXIST));
+        memberRelationRepository.delete(memberRelation);
+
+        Optional<MemberRelation> optionalMemberRelation =
+                memberRelationRepository.findBySourceIdAndTargetId(
+                        currentMember.getId(), targetMember.getId());
+        if (optionalMemberRelation.isPresent()) {
+            return new FollowerDeletedResponse(FollowStatus.FOLLOWING);
+        }
+        return new FollowerDeletedResponse(FollowStatus.NOT_FOLLOWING);
```

Comment:
```suggestion
		return optionalMemberRelation.isPresent()
				? FollowerDeletedResponse.of(FollowStatus.FOLLOWING)
				: FollowerDeletedResponse.of(FollowStatus.NOT_FOLLOWING);
```

삼항 연산자를 쓰면 더 직관적입니다!